### ISSUE TITLE
AUTH_REJECT error

### ISSUE THREAD

**Issue by Shulishin** [fjaros/wowchat#94]
Been attempting to log my bot in for stormforge's new netherwing realm however after successfully connecting to the server it attempts to authenticate for the realm and gets "AUTH_REJECT"

03:52:46.759 INFO  w.g.GameConnector - Connecting to game server Netherwing (141.94.138.34:8091)
03:52:47.090 INFO  w.g.GamePacketHandlerTBC - Connected! Authenticating...
03:52:47.781 ERROR w.g.GamePacketHandlerTBC - Failed to login to game server! Error code: AUTH_REJECT

Wondering if this is a server side issue (perhaps due to being launch day) or something identifiable and fixable on my end? Presumably though since it can authenticate with the server login details there's probably nothing to do for my end.

**Comment by fjaros** [comment]
Yeah. Unfortunately it seems Netherwing does not support platform=Mac . Unfortunately they also have warden enabled so platform=Windows will only work for 2 minutes and then you will get disconnected.. and eventually your account will probably be banned for failing to respond to enough warden messages.

Effectively this means the bot won't work on this realm. Your only option is to talk to the admins and try to get them to enable Mac logins.

**Comment by Shulishin** [comment]
Ah okay, makes sense. Thank you.
